code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/legionnaire.dm

```diff
@@ -323,10 +323,10 @@
 	var/next_use_time
 
 /obj/item/crusher_trophy/legionnaire_spine/effect_desc()
-	return "mark detonation to have a <b>[bonus_value]%</b> chance to summon a loyal legion skull"
+	return "mark detonation to have a <b>[bonus_value]%</b> chance to summon a loyal legion skull."
 
 /obj/item/crusher_trophy/legionnaire_spine/on_mark_detonation(mob/living/target, mob/living/user)
-	if(!rand(1, 100) <= bonus_value || target.stat == DEAD)
```

Comment:
I am going to bet that this is evaluating as
```DM
if((!rand(1, 100)) <= bonus_value || target.stat == DEAD)
    return
```

Reply:
Github please I clicked on 'single comment' can you just let me just comment on the code _without_ making it a review for once.

Reply:
it totally is yes. v dumb, should be ashamed, etc etc

Reply:
ill add it to the changelog then